fastlane/metadata/es-MX/description.txt

```diff
@@ -0,0 +1,12 @@
+Crea y gestiona tu web o blog WordPress directamente desde tu dispositivo iOS: crea y edita cualquier entrada o página, sube tus fotos y videos favoritos, accede a las estadísticas o responde a los comentarios fácilmente.
+ 
+Con WordPress para iOS, podrás publicar todo tu contenido desde la palma de tu mano. Escribe desde tu sofá el borrador de ese haiku que no te quitas de la cabeza. Sube y comparte en un momento esa increíble foto que hiciste durante el descanso en el trabajo. Responde a los últimos comentarios que te hayan dejado o descubre a qué nuevos países ha llegado hoy tu contenido echándole un vistazo a la sección de estadísticas.
+ 
+WordPress para iOS es un proyecto de código abierto, lo que significa que tú también puedes participar en su desarrollo. Descubre más información en https://apps.wordpress.com/contribute/.
+ 
+WordPress para iOS es compatible con sitios de WordPress.com y sitios de WordPress.org autoalojados que tengan WordPress 4.0 o superior.
+
+¿Necesitas ayuda con la aplicación? Entra en el foro de ayuda https://ios.forums.wordpress.org/ o déjanos un tweet en @WordPressiOS.
+
+Mira el aviso de privacidad para usuarios de California en https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.
+
```

Comment:
This and the other `es-MX` files got deleted [because of a failed download during the 19.8 finalization](https://github.com/wordpress-mobile/WordPress-iOS/pull/18612/). This time around, the download was successful so here they are again.